sw/device/sca/aes_serial.c

```diff
@@ -776,8 +776,13 @@ bool test_main(void) {
     LOG_INFO("Initializing entropy complex.");
     CHECK_STATUS_OK(aes_testutils_masking_prng_zero_output_seed());
     CHECK_DIF_OK(dif_aes_trigger(&aes, kDifAesTriggerPrngReseed));
+    bool idle = false;
+    do {
+      CHECK_DIF_OK(dif_aes_get_status(&aes, kDifAesStatusIdle, &idle));
+    } while (!idle);
   }
 #endif
+  CHECK_DIF_OK(dif_aes_trigger(&aes, kDifAesTriggerDataOutClear));
```

Comment:
I notice that the equivalent line in `aes_sca.c` is skipped for EnglishBreakfast. Is putting this line outside of the if/endif intentional?

Reply:
Yes it is. Only `aes_serial.c` runs on English Breakfast (where we use a different way to 0 masks because there is no CSRNG). `aes_sca.c` only runs on the actual chip and the CW310/340.